packages/composer.rb

```diff
@@ -14,7 +14,7 @@ class Composer < Package
   def self.preinstall
     if Dir.exists?("#{HOME}/.config") && !File.symlink?("#{HOME}/.config")
       # Save any existing configuration
-      FileUtils.cp_r "#{HOME}/.config", "#{CREW_PREFIX}" unless Dir.empty? "#{HOME}/.config"
+      FileUtils.cp_r "#{HOME}/.config", "#{CREW_PREFIX}", remove_destination: true unless Dir.empty? "#{HOME}/.config"
```

Comment:
This sounds dangerous.  What if there is config from other packages?

Reply:
It actually crashes without this option. On my machine, ~/.config is a symbolic link pointing to /usr/local/.config. It contains several configs for other packages. This line seems to use a cache or something like this since the ruby's doc says it should act like a cp -rf... It indeed works on my machine, and yes, with a bit of magic to me ...

Reply:
huh! this is the same on my machine:
```
chronos@localhost ~ $ ls -aFl .config
lrwxrwxrwx. 1 chronos chronos 22 Oct 16 00:53 .config -> /usr/local/.config/
```

Does chromebrew make that symlink? Or is that made by something else?

Reply:
The `xdg_base` package adds the symlink.  The reason we need to do this is some configs store executables here, otherwise the home directory is mounted noexec and these binaries will not execute.

Reply:
No pb to me with the symlink. I was forced to do this since composer crashed without this because of already existing files. The `remove_destination` option is said to act like the `force` option of a bash rm command :https://www.rubydoc.info/stdlib/fileutils/FileUtils#cp_r-class_method